test/integration/basic_excluded_prefixes_test.go

```diff
@@ -33,7 +33,7 @@ func TestExcludePrefixCheck(t *testing.T) {
 	nodesCount := 1
 
 	variables := map[string]string{
-		nsmd.ExcludedPrefixesEnv: "10.20.1.0/24",
+		nsmd.ExcludedPrefixesEnv:     "200.20.1.0/24",
```

Comment:
Could we use a prefix from 172.16/12 (as noted in [RFC 1918](https://tools.ietf.org/html/rfc1918)).

I think we probably want to continue using private IPs here, but it looks like Google is only drawing from 10.0.0.0/8, so we could avoid the collision that way.  Thoughts?

Reply:
That's good point, but I'm not sure about other clouds. Since our tests are fine on Azure, probably they don't use 10.0.0.0/8. But I don't mind to try 172.16/12 and figure out :)

Reply:
Could we have the ipam pick based on avoiding the exclude prefixes?